pkg/remotes/docker/opts.go

```diff
@@ -0,0 +1,61 @@
+package docker
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/remotes/docker"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// NewResolver returns a new resolver to a Docker registry
+func NewOpts(options *docker.ResolverOptions) *docker.ResolverOptions {
+	if options.Tracker == nil {
+		options.Tracker = docker.NewInMemoryTracker()
+	}
+
+	if options.Headers == nil {
+		options.Headers = make(http.Header)
+	}
+
+	resolveHeader := http.Header{}
+	if _, ok := options.Headers["Accept"]; !ok {
+		// set headers for all the types we support for resolution.
+		resolveHeader.Set("Accept", strings.Join([]string{
+			images.MediaTypeDockerSchema2Manifest,
+			images.MediaTypeDockerSchema2ManifestList,
+			ocispec.MediaTypeImageManifest,
+			ocispec.MediaTypeImageIndex, "*/*"}, ", "))
```

Comment:
Do you need to add oras artifact manifest?

Reply:
Not here